sdks/unreal/Agones/README.md

```diff
@@ -1,49 +1,125 @@
-# Agones UE4 Plugin
+﻿# Agones Unreal SDK
 
-For installation and usage instructions, see
-[Unreal Engine Game Server Client Plugin](https://agones.dev/site/docs/guides/client-sdks/unreal/).
+Agones is a multilayer dedicated game server scaling and orchestration platform that can run anywhere Kubernetes can run.
 
-If you'd like to contribute to the plugin, see
-[CONTRIBUTING.md](/CONTRIBUTING.md)
+This is a SDK inspired by the REST API to the Agones sidecars that allows engineers to talk to the sidecar from either C++ or Blueprints.
 
-## Developer Information
+## Getting Started

```

Comment:
These user docs should really live in: https://agones.dev/site/docs/guides/client-sdks/unreal/

There is a guide on how to document upcoming features so that they can be hidden until the next release here:
https://agones.dev/site/docs/contribute/#documentation-for-upcoming-features